### PR TITLE
Enable add sql binding quickpick

### PR DESCRIPTION
**Pull request by lucyzhang929**
Reverting this PR: https://github.com/microsoft/azuredatastudio/pull/17340 with two new changes:
- added (preview) to command title
- added `resourceExtname` to when clause to hide the command for untitled files and fixes https://github.com/microsoft/azuredatastudio/issues/17571

### CODE REVIEW

File: extensions/sql-database-projects/package.json [hunk 1]
```diff
@@ -168,6 +169,10 @@
         "title": "%sqlDatabaseProjects.editProjectFile%",
         "category": "%sqlDatabaseProjects.displayName%"
       },
+      {
+        "command": "sqlDatabaseProjects.addSqlBinding",
+        "title": "%sqlDatabaseProjects.addSqlBinding%",
+        "category": "MS SQL"},
```
Comment (Charles-Gagnon) [inline extensions/sql-database-projects/package.json:175]: }, on separate line

File: extensions/sql-database-projects/package.json [hunk 2]
```diff
@@ -268,6 +273,10 @@
           "command": "sqlDatabaseProjects.editProjectFile",
           "when": "false"
         },
+        {
+          "command": "sqlDatabaseProjects.addSqlBinding",
+          "when": "editorLangId == csharp && !azdataAvailable && resourceExtname"
```
Comment (Charles-Gagnon) [inline extensions/sql-database-projects/package.json:278]: `&& resourceScheme != untitled` would be better I think - files don't necessarily have to have extensions.

**Review by Charles-Gagnon — APPROVED** [review]

**Comment by Charles-Gagnon** [comment]
Let's also bump the version too so we're ready for the release (0.15.0)

File: extensions/sql-database-projects/package.json [hunk 3]
```diff
@@ -2,7 +2,7 @@
   "name": "sql-database-projects",
   "displayName": "SQL Database Projects",
   "description": "Enables users to develop and publish database schemas for MSSQL Databases",
-  "version": "0.14.0",
+  "version": "0.15.0",
```
Comment (kisantia) [inline extensions/sql-database-projects/package.json:5]: the version was already bumped to 0.14.0 after the October release, so this doesn't need to be bumped again. The current version in vscode is 0.13.1

Comment (lucyzhang929) [reply extensions/sql-database-projects/package.json:5]: Will revert the version, thanks!

**Review by kisantia — APPROVED** [review]

**Pull request merged**